src/field_impl.h

```diff
@@ -336,4 +336,32 @@ SECP256K1_INLINE static void secp256k1_fe_from_storage(secp256k1_fe *r, const se
 #endif
 }
 
+static void secp256k1_fe_impl_inv(secp256k1_fe *r, const secp256k1_fe *x);
+SECP256K1_INLINE static void secp256k1_fe_inv(secp256k1_fe *r, const secp256k1_fe *x) {
+#ifdef VERIFY
+    int input_is_zero = secp256k1_fe_normalizes_to_zero(x);
```

Comment:
Before this PR, the `secp256k1_fe_normalizes_to_zero` was done on an intermediate result (rather than the input `x`); would moving this check before the start of `impl_inv` change the behavior?